extension/dist/manifest.json

```diff
@@ -3,12 +3,18 @@
   "version": "1.0.1",
   "author": "marcbachmann",
   "description": "Enhances the `New Issue` button to support multiple templates.",
-  "homepage_url": "https://github.com/marcbachmann/github-issue-templates",
+  "homepage_url": "https:/github.com/marcbachmann/github-issue-templates",
+  "browser_action": {
+    "default_popup": "options.html"
+  },
   "manifest_version": 2,
   "minimum_chrome_version": "58",
   "permissions": [
     "https://github.com/",
+    "https://github.ibm.com/",
+    "https://*.github.ibm.com/",
     "https://raw.githubusercontent.com/",
+    "https://*.github.com/",
```

Comment:
Anyway, that didn't solve the issue.  Do you see also many errors to get assets and fonts in your browser?